Service Connector  Hub/test_func.py

```diff
@@ -0,0 +1,113 @@
+import os
+import gzip
+from io import BytesIO
+from func import handler
+from unittest import TestCase, mock
+
+
+def to_bytes_io(inp : str):
```

Comment:
Can we keep test related structure in separate `tests` directory. Any reason to move it out of there?

Reply:
I think it's much easier to run tests out-of-the-box if the test file and src file is in the same module. If they are not, I couldn't run the test straight away since the modules are different.

Reply:
Restored the structure to match dogweb for consistency, although losing the OOTB capability to run tests directly from Pycharm. However, there are some hacks that can be done to enable that.